File: ImageSelect/src/main/java/select/image/com/crop/CropCallBack.java
package select.image.com.crop;

import android.app.Activity;
import android.net.Uri;

/**
 * by y on 2017/4/7.
 */

public interface CropCallBack {
    Activity getCropActivity();

    void onCropCancel();

    void onCropSuccess(Uri uri);

    void onCropError(String errorMessage);

}

File: utils_imageselect/src/main/java/sample/util/develop/android/image/select/crop/CropCallBack.kt
package sample.util.develop.android.image.select.crop

import android.app.Activity
import android.net.Uri

interface CropCallBack {
    val cropActivity: Activity

    fun onCropCancel()

    fun onCropSuccess(uri: Uri?)

    fun onCropError(errorMessage: String?)
}
